xml/sle_update_online.xml

```diff
@@ -832,9 +832,10 @@ Continue? [y/n/? shows all options] (y):</screen>
     <para>
      Now add the modules you need for your installation.
     </para>
-<screen>&prompt.root;<command>SUSEConnect --list-extensions</command>
-[...]
-&prompt.root;<command>SUSEConnect -p sle-module-basesystem/15.1/x86_64</command></screen>
+<screen>
```

Comment:
The line break here is not correct IHMO.

Reply:
visually, it looks the same in PF and HTML, plus it is more convenient for multiline screen outputs 'cause if you want to manipulate with lines inside the &lt;screen/> tag, you dont have to cut off the markup itself as it is on separate lines